SpatialGDK/Source/SpatialGDK/Private/Interop/GlobalStateManager.cpp

```diff
@@ -601,6 +601,7 @@ void UGlobalStateManager::QueryGSM(const QueryDelegate& Callback)
 		}
 		else
 		{
+			ApplyVirtualWorkerMappingFromQueryResponse(Op);
```

Comment:
Why do we additionally apply this here (isn't distribution of this info already handled through replication of the component data)?

Reply:
It's my understanding that the client doesn't express interest in the client, and GSM data is only received by the client through the GSM entity query that happens on startup. I originally put logic in that would do as you've suggested but it never got received triggered, leading me to realise I had to put it here.